docs/advanced-features/automatic-static-optimization.md

```diff
@@ -20,7 +20,7 @@ If the above is not the case, Next.js will **statically optimize** your page aut
 
 During prerendering, the router's `query` object will be empty since we do not have `query` information to provide during this phase. After hydration, Next.js will trigger an update to your application to provide the route parameters in the `query` object.
 
-> **Note:** Parameters added with [dynamic routes](/docs/routing/dynamic-routes.md) to a page that's using [`getStaticProps`](/docs/basic-features/data-fetching.md#getstaticprops-static-generation) will always be available inside the `query` object.
+> **Note:** Parameters added with [dynamic routes](/docs/routing/dynamic-routes.md) to a page that's using [`getStaticProps`](/docs/basic-features/data-fetching/getStaticProps.md) will always be available inside the `query` object.
```

Comment:
All pages are currently lowercase, probably best to keep it that way 🤔

Reply:
Good call, will change file paths too to match. Note to self to change front PR with hash redirects

Reply:
@timneutkens looking at this now. The reason i chose camelCase was because `getInitialProps.md` was named that way. Do you still want me to change the file names? This will mean changing a large amount of internal links too